junction/proposals/management/commands/export_conf_proposals.py

```diff
@@ -0,0 +1,80 @@
+
+import io
+from os.path import join
+from settings.common import ROOT_DIR
+
+from django.shortcuts import get_object_or_404
+from django.core.exceptions import PermissionDenied
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+from xlsxwriter.workbook import Workbook
```

Comment:
It's better to export as CSV rather than XLSX.

XLSX needs special programs to run, whereas is plain text formatted data.

Reply:
+1 for csv

Reply:
Will do